### PR TITLE
A top-level version constant.

### LINKED ISSUE #229: Expose Version as a library-level constant
We should expose the current version of thriftrw as a library-level constant

### PR DESCRIPTION
**Pull request by bombela**
- Now we have a subpackage version with a constant Version.

Fixes #229

### CODE REVIEW

**Comment by prashantv** [comment]
Yeah I'd suggest a subpackage (either `thriftrw` or `version`)

**Comment by bombela** [comment]
I had a weird magical thing happening with go build, which likely pulled the binary over and over again from some sort of cache, so I ended up pushing here to confirm that I wasn't supposed to work!

**Comment by bombela** [comment]
Now we have version.Version ...

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ v0.4.0 (unreleased)
     types from values to types.
 -   Generate args structs and helpers for oneway functions.
 -   Expose whether a function is oneway to plugins.
+-   Library version is exposed via the subpackage version (version.Version).
```
Comment (breerly) [inline CHANGELOG.md]: Nit - The convention here is to use backticks, so `version.Version`.

Comment (breerly) [inline CHANGELOG.md]: Suggest renaming to:

- Expose the version of the library under `go.uber.org/thriftrw/version.Version`.

**Pull request merged**